app/blueprints/self_serve/routes.py

```diff
@@ -128,12 +98,31 @@ def view_form_questions():
     html = print_html(print_data)
     return render_template("view_questions.html", section_name=form_config["title"], question_html=html)
 
+@self_serve_bp.route("/section_questions", methods=["POST"])
+def view_section_questions():
+    # form_config = generate_form_config_from_request()
+    # print_data = generate_print_data_for_sections(
+    #     sections=[
+    #         {
+    #             "section_title": form_config["title"],
+    #             "forms": [{"name": form_config["form_id"], "form_data": form_config["form_json"]}],
+    #         }
+    #     ],
+    #     lang="en",
+    # )
+    # html = print_html(print_data)
+    # return render_template("view_questions.html", section_name=form_config["title"], question_html=html)
+    pass
 
-@self_serve_bp.route("build_section", methods=["GET", "POST"])
-def build_section():
+
+# CRUD routes
+
+# Create routes
+@self_serve_bp.route("create_section", methods=["GET", "POST"])
```

Comment:
Cant we use specific http method type for the crud operations refer the image 

![image](https://github.com/user-attachments/assets/579aeed1-1525-48e4-ae9b-16402a9fb616)

Reply:
based on this: https://restfulapi.net/resource-naming/#:~:text=2.4.%20Never%20use%20CRUD%20function%20names%20in%20URIs
 i will take out the CRUD from the uri and just set the methods